src/test-routes/formdata.ts

```diff
@@ -14,4 +15,11 @@ app.category("vanilla", () => {
       },
     };
   });
+  app.post("/formsdataurlencoded/pet/add", "UpdatePetWithForm", (request) => {
+    request.expect.containsHeader("content-type", "application/x-www-form-urlencoded");
+    request.expect.bodyEquals({ pet_type: "dog", pet_food: "meat", name: "Fido", status: "" });
```

Comment:
let's validate pet_id as well

Reply:
Good catch!

I wasn't correctly generating code to pass it!